jenkins/vs/buildimage-vs-image-test/Jenkinsfile

```diff
@@ -61,24 +61,22 @@ pipeline {
                     }
 				}
             }
-        }
 
-    }
-    post {
+            post {
+                always {
+                    junit(allowEmptyResults: true, keepLongStdio: true, testResults: 'sonic-mgmt/tests/logs/**/*.xml')
+                    archiveArtifacts(artifacts: 'sonic-mgmt/tests/logs/**')
+                }
 
-        always {
-            archiveArtifacts(artifacts: 'sonic-mgmt/tests/logs/**, sonic-mgmt/tests/results/**, kvmdump/**, ptfdump/**')
-            junit(allowEmptyResults: true, keepLongStdio: true, testResults: 'sonic-mgmt/tests/results/**/*.xml')
+                failure {
+                    archiveArtifacts(artifacts: 'kvmdump/**, ptfdump/**')
+                }
+            }
         }
 
-        fixed {
-            slackSend(color:'#00FF00', message: "Build job back to normal: ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)")
-            office365ConnectorSend(webhookUrl: "${env.SONIC_TEAM_TEST_WEBHOOK}")
-        }
-        regression {
-            slackSend(color:'#FF0000', message: "Build job Regression: ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)")
-            office365ConnectorSend(webhookUrl: "${env.SONIC_TEAM_TEST_WEBHOOK}")
```

Comment:
not sure why they are removed?

Reply:
I am going to merge this pr, but please add those notifications back in another pr.

Reply:
ack